hack/update-openapigen.sh

```diff
@@ -19,40 +19,24 @@ set -o nounset
 set -o pipefail
 
 if [[ -z "${GOPATH:-}" ]]; then
-  export GOPATH=$(go env GOPATH)
+    export GOPATH=$(go env GOPATH)
 fi
 
-PROJECT_ROOT=$(pwd)
-CODEGEN_PKG=$(pwd)/vendor/k8s.io/code-generator
-
-KATIB_VERSION_V1ALPHA3="v1alpha3"
-SWAGGER_CODEGEN_FILE_V1ALPHA3="pkg/apis/v1alpha3/swagger.json"
-
-KATIB_VERSION_V1BETA1="v1beta1"
-SWAGGER_CODEGEN_FILE_V1BETA1="pkg/apis/v1beta1/swagger.json"
-
+PROJECT_ROOT=${GOPATH}/src/github.com/kubeflow/katib
+CODEGEN_PKG=${PROJECT_ROOT}/vendor/k8s.io/code-generator
+VERSION_LIST=(v1alpha3 v1beta1)
 SWAGGER_VERSION="0.1"
```

Comment:
For each Katib version Swagger Version will be v0.1, is that fine?

Reply:
Is there any suggestion?

Reply:
@andreyvelich I've refined the schema of swagger version, which became `${katib_version}-{swagger_sub_version}`. So, with swagger version, we can figure out katib version who generated it.

Reply:
Thanks @sperlingxx, SGTM.